tests/integration_tests/base_tests.py

```diff
@@ -18,7 +18,7 @@
 """Unit tests for Superset"""
 
 from datetime import datetime
-import imp
+import importlib.util
```

Comment:
nit: I'd almost prefer to only import `find_spec`:
```suggestion
from importlib.util import find_spec
```